frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import pytest
+import shakedown
+
+
+@pytest.fixture(scope='session', autouse=True)
+def configure_universe(request):
+    stub_urls = {}
+
+    # prepare needed universe repositories
+    stub_universe_urls = os.environ.get('STUB_UNIVERSE_URL')
+    if not stub_universe_urls:
+        return
+    for url in stub_universe_urls.split(' '):
+        package_name = url.split('/')[4] # TODO: does this need to be a random string?
+        stub_urls[package_name] = url
+
+    # clear out the added universe repositores at testing end
+    def unconfigure_universe():
+        for name, url in stub_urls.items():
+            remove_package_cmd = 'dcos package repo remove {}'.format(name)
+            shakedown.run_dcos_command(remove_package_cmd)
+    request.addfinalizer(unconfigure_universe)
```

Comment:
I do it because, although prettier, using `yield` doesn't guarantee the teardown code runs. If the setup code throws an exception the teardown is skipped.